### PR TITLE
add blog post draft: redirect management in Sphinx

### PR DESCRIPTION
**Pull request by TimKam**

### CODE REVIEW

File: _posts/2017-07-26-managing-sphinx-redirects.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+When you use [Sphinx](http://www.sphinx-doc.org/en/stable/) to document your software, you sometimes move files to a different location as part of a content refactoring.
+Then, you need to set up redirects that point from the old to the new location of a content piece.
+If you make your documentation available on one particular domain, the obvious way to handle redirects is to ask the system administrator to configure the redirects on web server side, for example in Apache [.httaccess](https://httpd.apache.org/docs/current/howto/htaccess.html) files.
```
Comment (hilton) [inline _posts/2017-07-26-managing-sphinx-redirects.md]: `.htaccess`

Comment (hilton) [inline _posts/2017-07-26-managing-sphinx-redirects.md]: New paragraph for this sentence.

File: _posts/2017-07-26-managing-sphinx-redirects.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  <head>
+    <meta http-equiv="refresh" content="1; url=<./new_url>:" />
+    <script>
+      window.location.href = "<./new_url>"
+    </script>
+  </head>
+</html>
+```
+
+Every time you make a refactoring that moves an entire file, place the and HTML file with the template snippet to the folder you moved the source file from.
```
Comment (hilton) [inline _posts/2017-07-26-managing-sphinx-redirects.md]: Missing word after _place the_

File: _posts/2017-07-26-managing-sphinx-redirects.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+Of course, the HTML file needs to have the same name as the compiled moved file.
+For example, if your source file was `folders.rst`, name the output file `folders.html`.
+
+## Setup script in project configuration file
+By default, Sphinx won't process the redirect files (it only considers source files).
+That's why you need to write a couple of lines of Python code to move the redirect files to their correct place in the build directory.
```
Comment (hilton) [inline _posts/2017-07-26-managing-sphinx-redirects.md]: You say _move_ twice in this section, but I think you mean _copy_.

File: _posts/2017-07-26-managing-sphinx-redirects.md (hunk 2 above)
Comment (TimKam) [inline _posts/2017-07-26-managing-sphinx-redirects.md]: I need to change this sentence a bit.

File: _posts/2017-07-26-managing-sphinx-redirects.md (hunk 3 above)
Comment (TimKam) [inline _posts/2017-07-26-managing-sphinx-redirects.md]: Correct!

**Pull request merged**